apps/astarte_pairing_api/lib/astarte_pairing_api_web/router.ex

```diff
@@ -22,6 +22,13 @@ defmodule Astarte.Pairing.APIWeb.Router do
   pipeline :api do
```

Comment:
Since this is realm-scoped, let's call it `:realm_api` instead of `:api`.
```suggestion
  pipeline :realm_api do
```